README.md

```diff
@@ -8,33 +8,48 @@
+The workflow processes RNA-sequencing data from FastQ files. It runs quality control on the raw data ([FastQC](https://www.bioinformatics.babraham.ac.uk/projects/fastqc/)), detects fusion genes ([STAR-Fusion](https://github.com/STAR-Fusion/STAR-Fusion), [Fusioncatcher](https://github.com/ndaniel/fusioncatcher), [Ericscript](https://sites.google.com/site/bioericscript/), [Pizzly](https://github.com/pmelsted/pizzly), [Squid](https://github.com/Kingsford-Group/squid)), gathers information ([FusionGDB](https://ccsm.uth.edu/FusionGDB/index.html)), visualizes the fusions ([FusionInspector](https://github.com/FusionInspector/FusionInspector)), performs quality-control on the results ([MultiQC](http://multiqc.info)) and finally generates custom summary report.
+
+![Final summary report](docs/images/example-summary-report.png)
 
-**Note: Make sure to read the installation guide before running the pipeline.**
+The pipeline works with both single-end and paired-end data, though not all fusion detection tools work with single-end data (Ericscript, Pizzly, Squid and FusionInspector).
 
 The pipeline is built using [Nextflow](https://www.nextflow.io), a workflow tool to run tasks across multiple compute infrastructures in a very portable manner. It comes with docker / singularity containers making installation trivial and results highly reproducible.
 
+| Tool            | Single-end reads | CPU (recommended) | RAM (recommended) |
+| --------------- |:----------------:|:-----------------:|:-----------------:|
+| [Star-Fusion](https://github.com/STAR-Fusion/STAR-Fusion/wiki) | Yes              | >=16 cores        | ~30GB             |
+| [Fusioncatcher](https://github.com/ndaniel/fusioncatcher/blob/master/doc/manual.md) | Yes              | >=16 cores        | ~60GB             |
+| [Ericscript](https://sites.google.com/site/bioericscript/getting-started) | **No**           | >=16 cores        | ~30GB             |
+| [Pizzly](https://github.com/pmelsted/pizzly) | **No**           | >=16 cores        | ~30GB             |
+| [Squid](https://github.com/Kingsford-Group/squid) | **No**           | >=16 cores        | ~30GB             |
+| [FusionInspector](https://github.com/FusionInspector/FusionInspector/wiki) | **No**           | >=16 cores        | ~30GB             |
+
+> **TL;DR:** Make sure to download all required references for each tool. More details can be found in section [tools](docs/tools.md).
+
 ```bash
 nextflow run nf-core/rnafusion --reads '*_R{1,2}.fastq.gz' --genome GRCh38 -profile docker --star_fusion --fusioncatcher --ericscript --pizzly --squid --fusion_inspector
```

Comment:
Can we set up the pipeline so that if _none_ of the fusion tools are specified on the command line, then they are all used? Or perhaps `--skip_*` parameters turn off tools?

I'm just thinking that most of the time people will want to run most/all of the tools.

Reply:
We could but we would have to be a little more specific. For example, right now we support only one visualization tool but I would like to implement in next version other tools like Clicker (#25). Sooo we could make a parameter `all-fusion-tools` or something like that, which would only run tools that detect the fusion genes only. Comments?

Reply:
After a talk, we are ignoring this.